libs/template/templates/default-python/template/{{.project_name}}/res/job.yml.tmpl

```diff
@@ -0,0 +1,34 @@
+resources:
+  jobs:
+    {{.project_name}}-job:
+      name: {{.project_name}}-job
+      schedule:
+        quartz_cron_expression: "44 37 8 * * ?"
+        timezone_id: Europe/Amsterdam
+      job_clusters:
+        - job_cluster_key: job-cluster
+          new_cluster:
+            spark_version: 12.2.x-scala2.12
+            node_type_id: {{smallest_node_type}}
+            num_workers: 1
+      tasks:
+        - task_key: notebook-task
+          job_cluster_key: job-cluster
+          notebook_task:
+            notebook_path: ../src/notebook-sample.py
+        - task_key: python-wheel-task
+          depends_on:
+            - task_key: notebook-task
+          job_cluster_key: job-cluster
+          python_wheel_task:
+            package_name: "{{.project_name}}"
+            entry_point: "run"
+          libraries:
+          {{- /* TODO: this path is wrong, it should say ../dist/*.whl */ -}}}}
+          - whl: dist/*.whl
```

Comment:
@lennartkats-db I think we actually thrive to have paths in bundle config relative to bundle root (as you can see here https://github.com/databricks/bundle-examples/pull/1/files) and not the file that uses it
@pietern any thoughts?

Reply:
We need to get to consistency either way. The way it works for notebooks has been in place for a long time and has been by design. But we could change it, especially if we think our recommended DAB structure is that notebooks and source code lives in a different directory than job definitions.

@fjakobs any thoughts on this?

Reply:
+1 for consistency. We can either have paths always relative to the root or always relative to the config file. Both ways are fine as long as we are consistent and not mix them.

Reply:
@andrewnester  It's hard to say exactly which one is better. But I suppose with the file extensions, it really feels more like a file path than a package path. Which would suggest sticking to `notebook_path: ../src/notebook.ipynb` is preferable.

Reply:
@lennartkats-db I agree with unifying the paths resolution but also not sure what's the preferred mode is. As a contrary example, we recently introduce new include/exclude section for sync which allows to define glob patterns for including/excluding files to sync. These patterns are currently defined based on the bundle root.

https://github.com/databricks/cli/pull/671

I guess it's best to have a short discussion with myself, you and @pietern to align on this.

Reply:
As long as these configuration files are located in the bundle root, then it doesn't matter because they are equivalent. Only when folks start to put configuration files in nested folders this starts to matter. For the notebook path case, we figured it was weird to anchor it at the bundle root, if both the notebook and configuration are located in a nested directory (e.g. a file at `foo/bar.yml` that would refer to `foo/notebook.py`). It should refer to `./notebook.py`. This also makes the directory relocatable without modifying the configuration.

Because of this, I believe we should anchor every path at the directory of the file it is defined in.

We can do the same for the sync include paths, though it only makes sense if we allow configuring them in nested configuration as well (and that we properly merge/append additional include/exclude definitions).

Reply:
@andrewnester If you agree with above, could you take a pass at trying to make paths relative to the directory of the config file they are defined in?

Reply:
New PR related to the above: https://github.com/databricks/cli/pull/708